src/applications/personalization/profile/tests/e2e/profile.direct-deposit-gating.cypress.spec.js

```diff
@@ -4,48 +4,82 @@ import { PROFILE_PATHS } from '../../constants';
 import mockUserNotInEVSS from '../fixtures/users/user-non-vet.json';
 import mockUserInEVSS from '../fixtures/users/user-36.json';
 
-import mockPaymentInfoNotEligible from '../fixtures/payment-information/direct-deposit-is-not-eligible.json';
-import mockPaymentInfoIncompetent from '../fixtures/payment-information/direct-deposit-incompetent.json';
-import mockPaymentInfoDeceased from '../fixtures/payment-information/direct-deposit-deceased.json';
-import mockPaymentInfoFiduciary from '../fixtures/payment-information/direct-deposit-fiduciary.json';
+import mockDD4CNPNotEligible from '../fixtures/dd4cnp/dd4cnp-is-not-eligible.json';
+import mockDD4CNPNotEnrolled from '../fixtures/dd4cnp/dd4cnp-is-not-set-up.json';
+import mockDD4CNPEnrolled from '../fixtures/dd4cnp/dd4cnp-is-set-up.json';
+import mockDD4CNPIncompetent from '../fixtures/dd4cnp/dd4cnp-incompetent.json';
+import mockDD4CNPDeceased from '../fixtures/dd4cnp/dd4cnp-deceased.json';
+import mockDD4CNPFiduciary from '../fixtures/dd4cnp/dd4cnp-fiduciary.json';
+
+import mockDD4EDUEnrolled from '../fixtures/dd4edu/dd4edu-enrolled.json';
+import mockDD4EDUNotEnrolled from '../fixtures/dd4edu/dd4edu-not-enrolled.json';
+
+// TODO: remove this when we are no longer gating DD4EDU with a feature flag
+const dd4eduEnabled = {
+  data: {
+    type: 'feature_toggles',
+    features: [
+      {
+        name: 'ch33_dd_profile',
+        value: true,
+      },
+    ],
+  },
+};
+
+function confirmDDBlockedAlertIsNotShown() {
+  cy.findByText(/You can’t update your financial information/i).should(
+    'not.exist',
+  );
+}
+
+function confirmDDBlockedAlertIsShown() {
+  cy.findByText(/You can’t update your financial information/i)
+    .should('exist')
+    .closest('.usa-alert-error')
+    .should('exist');
+}
+
+function confirmDirectDepositIsAvailable() {
+  // the DD item should exist in the sub nav
+  cy.findByRole('navigation', { name: /secondary/i }).within(() => {
+    cy.findByRole('link', { name: /direct deposit/i }).should('exist');
+  });
 
-describe('Direct Deposit', () => {
-  function confirmDDBlockedAlertIsNotShown() {
```

Comment:
I lifted these first three helper functions out of the `describe` block. I don't know the best practice here, but it seems like these reused helpers shouldn't live here. 🤷‍♂️